source/entrance.cpp

```diff
@@ -436,6 +442,165 @@ static void ShuffleEntrancePool(std::vector<Entrance*>& entrancePool, std::vecto
   }
 }
 
+std::unordered_map<s16, std::string> entranceNames = {
+  { 0x0000, "KF Deku Tree Entrance" },
+  { 0x0209, "Deku Tree" },
+  { 0x0004, "DMT Dodongo's Cavern Entrance" },
+  { 0x0242, "Dodongo's Cavern" },
+  { 0x0028, "ZF Jabu Jabu's Belly Entrance" },
+  { 0x0221, "Jabu Jabu's Belly" },
+  { 0x0169, "SFM Forest Temple Entrance" },
+  { 0x0215, "Forest Temple" },
+  { 0x0165, "DMC Fire Temple Entrance" },
+  { 0x024A, "Fire Temple" },
+  { 0x0010, "LH Water Temple Entrance" },
+  { 0x021D, "Water Temple" },
+  { 0x0082, "Colossus Spirit Temple Entrance" },
+  { 0x01E1, "Spirit Temple" },
+  { 0x0037, "Graveyard Warp Pad" },
+  { 0x0205, "Shadow Temple" },
+  { 0x0098, "Kakariko Bottom of the Well Entrance" },
+  { 0x02A6, "Bottom of the Well" },
+  { 0x0088, "ZF Ice Cavern Entrance" },
+  { 0x03D4, "Ice Cavern" },
+  { 0x0008, "GF Training Grounds Entrance" },
+  { 0x03A8, "Gerudo Training Grounds" },
+  { 0x0433, "KF from Mido's House" },
```

Comment:
You don't have to change this, but I feel like it'd make more sense to use `to` instead of `from`, as what it says now implies the entrance that's inside Mido's House that leads to KF. Same with a lot (if not all) other names that say `from`.

Reply:
I'm half and half on it, 'cause technically it's not wrong (the entrance is "[arriving in] Kokiri forest from Mido's House") but you're right that some of them sound weird written that way. I'm kind of torn between choosing whatever sounds the most straightforward for every entrance, or keeping them all in a semi-consistent format.
Could maybe go the other way, and use `[origin] to [destination]` for all of them?